documentation/book/assembly-distributed-tracing.adoc

```diff
@@ -0,0 +1,64 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-distributed-tracing-{context}']
+= Distributed tracing
+
+{ProductName} provides support for the distributed tracing of transactions within a typical Kafka architecture. When Kafka client applications are instrumented for distributed tracing, {ProductName} generates __traces__ for transactions such as producing and consuming messages, or writing offsets to the consumer offsets log.
+
```

Comment:
I suggest we add a screenshot of the Jaeger UI here. Can you supply this, @scholzj ?

Reply:
If you search through Google Hangouts you might find some screenshots there. Otherwise I can of course make some, but probably only after I get back home.

Reply:
I will get you a screenshot over the weekend when finishing my Tracing blog post.

Reply:
Many thanks!